code.js

```diff
@@ -0,0 +1,54 @@
+function mkSet(parent,rank,n) { 
+  //function to initialize parent[] and rank[]
+	for(let i=0;i<n;i++) { 
+		parent[i]=i;
+		rank[i]=0;
+	} 
+} 
+
+function findParent(parent,component) { 
+	//function is to find the root of the sets
+	if(parent[component]==component) 
+		return component; 
+
+	return parent[component] = findParent(parent,parent[component]); 
+} 
+// merge two disjoint sets based on their ranks.
+function unionSet(u, v, parent, rank) {
+	if(rank[u]<rank[v]) {
+		parent[u]=v;
+	} else if(rank[v]<rank[u]) {
+		parent[v]=u;
+	} else{
+		parent[v]=u; 
+		rank[u]++;//since the rank increases if the ranks of two sets are same 
+	} 
+} 
+
+function kruskalAlgo(edge) {
```

Comment:
Can you please use a standard graph representation, i.e. an adjacency matrix or list?

Reply:
I changed it to an adjacency matrix.